conjur/client.py

```diff
@@ -104,10 +102,17 @@ def __init__(self,
                             **loaded_config)
             self._api.login(login_id, password)
         else:
-            logging.info("Using API key with netrc credentials...")
+            try:
+                loaded_netrc = CredentialsFromFile.load()
+
+            except Exception:
+                # pylint: disable=line-too-long, raise-missing-from
+                raise RuntimeError("Unable to authenticate with Conjur. Please log in and try again")
```

Comment:
add `from ex`.

Reply:
added but why do we care to chain the exception if the in the end we get the same message?

Reply:
For the debug printing. Inner exception may contain the actual failure data necessary for investigation.